.gitignore

```diff
@@ -2,3 +2,4 @@ build
 *.pyc
 dist
 django_kronos.egg-info
+.idea/
```

Comment:
This directory is specific to your environment, so you should probably add it to your global ignore file and remove it from this one.